src/MARTI.php

```diff
@@ -38,12 +36,9 @@
 
 		$dice = new dice();
 
-		//check if all emails are registered
-		try {
-			$dice->checkIfMailsAreRegistered($output['emails']);
-		} catch (exception $e) {
-			exit($e->getMessage());
-		}
+		//check if all emails are registered.
+		// This method exits if one of them is not
+		$dice->requireMailsAreRegistered($output['emails']);
```

Comment:
In general, exiting from a "library" class like `dice` seems like bad form.  However, I see that it's been used in the past.  Maybe this is considered acceptable style in the PHP world?

Reply:
You'd be suprised how much is allowed in the PHP world 😅 
But I just wanted to make the code consistent in that regards. All other code snippets simply call exit when the result is not the desired one, so if we want to change this behaviour at some point we really should change it for all the code at once.